src/components/Application/ProgressBar.tsx

```diff
@@ -0,0 +1,120 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+import { Accordion, Container, Grid, Header, Label, List, Sticky } from 'semantic-ui-react'
+import { TemplateSectionPayload } from '../../utils/types'
+
+interface SectionPage {
+  sectionIndex: number
+  currentPage: number
+}
+
+interface ProgressBarProps {
+  serialNumber: string
+  templateSections: TemplateSectionPayload[]
+  sectionPage?: SectionPage
+}
+
+const ProgressBar: React.FC<ProgressBarProps> = ({
+  serialNumber,
+  templateSections,
+  sectionPage,
+}) => {
+  const childrenPanels = (section: string, totalPages: number) => {
+    const pages = Array.from(Array(totalPages).keys(), (n) => n + 1)
+    return (
+      <List>
+        {pages.map((number) => (
+          <List.Item
+            as={Link}
+            to={`applications/${serialNumber}/${section}/Page${number}`}
```

Comment:
```suggestion
            to={`/applications/${serialNumber}/${section}/Page${number}`}
```
This seems to do the trick for me. Not 100% clear why.

Reply:
same

Reply:
It works! Thank you so much